packages/generator/src/v1.1/wireObjectTypeV2ToV1ObjectInterfaceString.ts

```diff
@@ -34,33 +35,52 @@ ${
     ).join("\n")
   }
 
-/**
- * ${input.description}
- */
-export interface ${input.apiName} extends OntologyObject {
+  ${
+    getDescriptionIfPresent(input.description)
+  }export interface ${input.apiName} extends OntologyObject {
   readonly __apiName: "${input.apiName}";
   readonly __primaryKey: ${
     wirePropertyTypeV2ToTypeScriptType(
       input.properties[input.primaryKey].dataType,
     )
   };
 ${
-    Object.entries(input.properties).map((
+    Object.entries(input.properties).flatMap((
       [propertyName, propertyDefinition],
-    ) =>
-      `${
-        getDescriptionIfPresent(propertyDefinition.description)
-      }readonly ${propertyName}: ${
-        wirePropertyTypeV2ToTypeScriptType(propertyDefinition.dataType)
-      } | undefined`
-    ).join(";\n")
+    ) => {
+      const propertyType = wirePropertyTypeV2ToTypeScriptType(
+        propertyDefinition.dataType,
+      );
+      const entries = [
+        `${
+          getDescriptionIfPresent(propertyDefinition.description)
+        }readonly ${propertyName}: ${propertyType} | undefined`,
+      ];
+
+      if (isReservedKeyword(propertyName)) {
+        entries.push(`/** @deprecated */
+          readonly ${propertyName}_: ${propertyType} | undefined`);
+      }
+
+      return entries;
+    }).join(";\n")
   }
 ${
-    linkTypes.map(linkType =>
-      `readonly ${linkType.apiName}: ${
-        linkType.cardinality === "MANY" ? "MultiLink" : "SingleLink"
-      }<${linkType.objectTypeApiName}>`
-    ).join(";\n")
+    linkTypes.flatMap(linkType => {
+      const entries = [
+        `readonly ${linkType.apiName}: ${
+          linkType.cardinality === "MANY" ? "MultiLink" : "SingleLink"
+        }<${linkType.objectTypeApiName}>`,
+      ];
+
+      if (isReservedKeyword(linkType.apiName)) {
+        entries.push(`/** @deprecated */
```

Comment:
Should we explain why it is deprecated (it might be confusing if they use ontology primitives to mark a property is experimental or deprecated andthink this is related)

Reply:
Added some more details